src/pudl_archiver/frictionless.py

```diff
@@ -78,7 +79,10 @@ class DataPackage(BaseModel):
 
     @classmethod
     def from_filelist(
-        cls, name: str, files: list[DepositionFile], resources: dict[str, ResourceInfo]
+        cls,
+        name: str,
+        files: Iterable[DepositionFile],
```

Comment:
`mypy` was complaining about how `files.values()` wasn't a list. Instead of converting it to list I relaxed the type requirements.